go/vt/sqlparser/ast.go

```diff
@@ -136,8 +144,8 @@ type Insert struct {
 }
 
 func (node *Insert) Format(buf *TrackedBuffer) {
-	buf.Myprintf("insert %vinto %v%v %v%v",
-		node.Comments,
+	buf.Myprintf("insert %v%sinto %v%v %v%v",
```

Comment:
Okay, I've seen another place (AST_DISTINCT) that a node have trailing whitespace. My intuition is that node name will not have leading or trailing white spaces. It seems that, like you mentioned, some ast nodes will have trailing white spaces.